src/components/OcActionDrop.vue

```diff
@@ -60,20 +60,25 @@
  * While the authoring practices linked above advises to use `aria-controls` to create a programmatic relation between button and dropdown in reality screen reader support is close to non-existent. As of Fall 2019, only JAWS supports it, but has it disabled by default. The relation between trigger and dropdown should be communicated with the proximity of both in the DOM (see above).
  *
  * */
+import OcButton from "./OcButton.vue"
+
 export default {
   name: "OcActionDrop",
   status: "review",
   release: "1.0.0",
+  components: {
+    OcButton,
+  },
   props: {
     /**
      * Supplies button variation to give additional meaning.
      * `primary, danger`
      */
     buttonVariation: {
       type: String,
-      default: "default",
+      default: "passive",
```

Comment:
default is not a valid prop-value for oc-button variation prop

Reply:
how can we add tests, so that it does not happen again?

Reply:
validator used here and in the `OcButton` component will throw `vue [warn]` if the invalid prop is used.
we've to catch the console warning and test if waring for invalid props are not present, like in https://github.com/owncloud/owncloud-design-system/pull/1364/files where invalid prop vue warns are thrown as real error -> after merge we could use that before every test cases

Reply:
Could you also add all other valid button variations? can be found here https://github.com/owncloud/owncloud-design-system/blob/64d5456127a30a6db5744cae9d0718fb1875476d/src/components/OcButton.vue#L93

Reply:
fixed